tests/acceptance/test_organization_integration.py

```diff
@@ -0,0 +1,150 @@
+            },
+        )
+        self.org_integration = self.model.add_organization(self.org, self.user)
+        self.login_as(self.user)
+
+        self.integration_settings_path = 'sentry-api-0-organization-integrations'
+
+    def create_user_with_role(self, role, organization=None, teams=None):
+        """
+        Owner, manager, admin
+        """
+        user = self.create_user()
+        self.create_member(
+            organization=organization if organization else self.organization,
+            user=user,
+            role=role,
+            teams=teams if teams else [],
+        )
+        return user
+
+    def load_page(self, url):
+        self.browser.get(url)
+        self.browser.wait_until_not('.loading-indicator')
+
+    def assert_button_element(self, button, label, disabled=False, icon=None):
+        disabled = 'true' if disabled else 'false'
+        assert button.get_disabled() == disabled
+        assert button.get_label() == label
+        if icon:
+            assert button.get_icon_href() == icon
+
+
+class OrganizationIntegrationSettingsTest(OrganizationIntegrationAcceptanceTestCase):
+    """
+    As a user(type?), I can setup, configure, and remove an integration.
+    """
+    # TODO(lb): Tests to be written
+    # test_setup_new_integration_with_repository
+    # test_setup_new_integration_with_issue_sync
+    # test_remove_existing_integration_installation
+    # test_update_legacy_integration
+    # test_user_permissions_for_integration_settings
+    # test_add_multiple_integrations_to_one_provider
+
+    def setUp(self):
+        super(OrganizationIntegrationSettingsTest, self).setUp()
+        self.org_integration_settings_path = u'/settings/{}/integrations/'.format(
+            self.organization.slug)
+
+        self.provider = mock.Mock()
+        self.provider.key = 'example'
+        self.provider.name = 'Example Installation'
+
+    def assert_can_create_new_installation(self):
+        self.load_page(self.org_integration_settings_path)
+        org_settings_page = OrganizationIntegrationSettingsPage(
+            browser=self.browser
+        )
+        provider_element = org_settings_page.get_provider(self.provider)
+
+        integration_details_modal = org_settings_page.click_install_button(provider_element)
+        integration_details_modal.add_button.click()
+        org_settings_page.click_through_integration_setup(
+            integration_details_modal,
+            ExampleIntegrationSetupWindowElement,
+            {'name': self.provider.name}
+        )
+
+        installation_element = provider_element.get_installation_with_name(self.provider.name)
+        assert installation_element
+        assert Integration.objects.filter(
+            provider=self.provider.key,
+            external_id=self.provider.name
+        ).exists()
+
+        # TODO(lb): check issues details page and see that integration shows in linked issues
+        return installation_element
```

Comment:
I would make that process a separate test. You could setup state in the database, and then view issue details to ensure the integration links show up.

Reply:
I agree, but I'd rather get this PR out first. And I can follow up by checking things off the list above. Don't want to add more, until I'm solid on the page object abstraction